argus.yaml

```diff
@@ -120,10 +120,27 @@ authHeader: ["dXNlcjpwYXNz"]
 # request is a config section related to operation authorization
 # and request validation.
 request:
```

Comment:
I'm open to suggestions! I think it would just be good to find a way to hierarchically group things together when it makes sense. We can address them in future PRs as well.